### ISSUE TITLE
Scala trace propagation lost over a play 2.5 BodyParser

### ISSUE THREAD

**Issue by htmldoug** [DataDog/dd-trace-java#432]
## Repro
Clone failing project: https://github.com/htmldoug/datadog-netty4-failing/tree/master/2-play-scala-seed

```
cd 2-play-scala-seed
sbt stage
cd target/universal/stage
java -cp 'conf/:lib/*' "-Ddd.service.name=datadog_test_app" "-Ddd.trace.span.tags=env:dev" "-Ddd.integration.netty.enabled=true" "-Ddd.writer.type=LoggingWriter" "-Ddd.priority.sampling=true" -javaagent:~/.ivy2/cache/com.datadoghq/dd-java-agent/jars/dd-java-agent-0.13.0-SNAPSHOT.jar play.core.server.ProdServerStart
```

```
curl -X POST -d 'hai' localhost:9000/post
```

Expected: One `trace_id` for this request
Actual: Two separate `trace_id` values.
```
[application-akka.actor.default-dispatcher-7] INFO datadog.trace.agent.common.writer.LoggingWriter - write(trace): [{"type":"web","error":0,"meta":{"http.status_code":"200","component":"play-action","span.kind":"server","http.url":"/post","env":"dev","thread.name":"application-akka.actor.default-dispatcher-7","http.method":"POST","thread.id":"42","span.type":"web"},"metrics":{"_sample_rate":1.0,"_sampling_priority_v1":1},"duration":495080,"name":"play.request","resource":"POST /post","service":"datadog_test_app","trace_id":3394962754312157747,"span_id":4291744685937419745,"start":1533693728276018428,"parent_id":0}]
[netty-event-loop-6] INFO datadog.trace.agent.common.writer.LoggingWriter - write(trace): [{"type":"web","error":0,"meta":{"http.status_code":"200","component":"netty","span.kind":"server","http.url":"http://localhost:9000/post","peer.hostname":"localhost","env":"dev","peer.port":"63781","thread.name":"netty-event-loop-6","http.method":"POST","thread.id":"27","span.type":"web"},"metrics":{"_sample_rate":1.0,"_sampling_priority_v1":1},"duration":9722625,"name":"netty.request","resource":"POST /post","service":"datadog_test_app","trace_id":2725413885196180439,"span_id":199824513983472727,"start":1533693728268121976,"parent_id":0}]
```

Formatted:
```json
[
  {
    "type": "web",
    "error": 0,
    "meta": {
      "http.status_code": "200",
      "component": "play-action",
      "span.kind": "server",
      "http.url": "/post",
      "env": "dev",
      "thread.name": "application-akka.actor.default-dispatcher-7",
      "http.method": "POST",
      "thread.id": "42",
      "span.type": "web"
    },
    "metrics": {
      "_sample_rate": 1.0,
      "_sampling_priority_v1": 1
    },
    "duration": 495080,
    "name": "play.request",
    "resource": "POST /post",
    "service": "datadog_test_app",
    "trace_id": 3394962754312157747,
    "span_id": 4291744685937419745,
    "start": 1533693728276018428,
    "parent_id": 0
  }
]
[
  {
    "type": "web",
    "error": 0,
    "meta": {
      "http.status_code": "200",
      "component": "netty",
      "span.kind": "server",
      "http.url": "http://localhost:9000/post",
      "peer.hostname": "localhost",
      "env": "dev",
      "peer.port": "63781",
      "thread.name": "netty-event-loop-6",
      "http.method": "POST",
      "thread.id": "27",
      "span.type": "web"
    },
    "metrics": {
      "_sample_rate": 1.0,
      "_sampling_priority_v1": 1
    },
    "duration": 9722625,
    "name": "netty.request",
    "resource": "POST /post",
    "service": "datadog_test_app",
    "trace_id": 2725413885196180439,
    "span_id": 199824513983472727,
    "start": 1533693728268121976,
    "parent_id": 0
  }
]
```

**Comment by htmldoug** [comment]
https://github.com/lucidsoftware/opentracing-playframework/pull/2 never got that part working.
I can confirm it does work in Kamon with kamon-play-2.5 and kamon-akka-2.5 0.6.7 on the classpath. There may be some possible inspiration there.

**Comment by realark** [comment]
Fix shipped in 0.16.0
